### ISSUE TITLE
Add regression test for starting wish activity

### ISSUE THREAD

**Issue by ironjan** [ironjan/metal-only#127]
After #125, wish activity can always be started. Add refrain test so that this is checked (it had crash to open after switch to AA).

Also do this for donation, feedback, and about if not already done (feedback is not active but intent).

**Comment by ironjan** [comment]
This is already done, except for checking the feedback intent (see #132).
